dspace-api/src/main/java/org/dspace/discovery/SolrServiceImpl.java

```diff
@@ -462,10 +467,13 @@ public void cleanIndex(boolean force) throws IOException,
             }
             if (force)
             {
-                getSolr().deleteByQuery("search.resourcetype:[2 TO 4]");
+                getSolr().deleteByQuery(RESOURCE_TYPE_FIELD + ":[2 TO 4]");
```

Comment:
Should this perhaps also use the type constants rather than hardcoding them? Just a minor nitpick.